src/elisp/treemacs-visuals.el

```diff
@@ -240,7 +247,12 @@ inserted into `treemacs-icons-hash'."
           treemacs-icon-tag-leaf-text        (cdr (assq 'treemacs-icon-tag-leaf-text        treemacs--defaults-icons))))
   (treemacs--create-icons)
   (when (fboundp 'clear-image-cache)
-    (clear-image-cache)))
+    (clear-image-cache t))
+  (dolist (buf (buffer-list))
```

Comment:
No need to iterate over the whole list. You'll find a frame -> buffer alist in `treemacs--buffer-access`.

Reply:
Fantastic. `treemacs-run-in-every-buffer` makes it a lot easier, indeed. I changed the logic to use it.